README.md

```diff
@@ -165,12 +165,22 @@ library
   default-language:    Haskell2010
 ```
 
-If you want to be able to import `Extra.*` modules when using `mixins` approach,
-you need to list those modules under `mixins` field as well, like this:
+This way, you can't access any other module of relude. If you want to use e.g.
+`Relude.Extra` (which reexports all `Relude.Extra.*` modules), you can add it
```

Comment:
I think that it's okay to leave `Relude.Extra.Enum` as the example to show that you can use just some of the components of the `Extra.*` modules.
Though maybe it would be worth mentioning that there is `Relude.Extra` which reexports all extra modules, however, this is not necessary to be done in the mixins section :slightly_smiling_face: